tests/general.rs

```diff
@@ -253,10 +253,8 @@ fn test_Xpsk0_expected_value() {
 
 #[test]
 #[cfg(feature = "hfs")]
-#[cfg(feature = "pqclean_kyber1024")]
+#[cfg(feature = "nist3_kyber1024")]
 fn test_NNhfs_sanity_session() {
-    // Due to how PQClean is implemented, we cannot do deterministic testing of the protocol.
-    // Instead, we will see if the protocol runs smoothly.
```

Comment:
Why remove this comment? liboqs also incorporates its own random number generator, so this comment is still applicable.

Reply:
ah okay i'll put it back